float64 latitude
float64 longitude
---
float64[] intensity
float64 latitude
float64 longitude